### ISSUE TITLE
Tab switcher not working on page where multiple exist

### ISSUE THREAD

**Issue by qwertytech** [uikit/uikit#2324]
I'm currently trying to use two tab switchers on a single page but when I do, neither work and my un-nav all start expanded.

```
<div uk-grid>
    <div class="uk-width-auto@m">
        <ul class="uk-tab-left" uk-tab="connect: #component-tab-left; animation: uk-animation-fade">
            <li><a href="#">Active</a></li>
            <li><a href="#">Item</a></li>
            <li><a href="#">Item</a></li>
        </ul>
    </div>
    <div class="uk-width-expand@m">
        <ul id="component-tab-left" class="uk-switcher">
            <li>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</li>
            <li>Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</li>
            <li>Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur, sed do eiusmod.</li>
        </ul>
    </div>
</div>


<div id="itemPush" uk-offcanvas="mode: push; flip: true">
    <div class="sidebar-right uk-offcanvas-bar">
        <ul uk-tab="connect: #item-tabs">
            <li class="uk-active"><a href="#">Item 1</a></li>
            <li><a href="#">Item 2</a></li>
            <li><a href="#">Item 3</a></li>
            <li><a href="#">Item 4</a></li>
        </ul>
        <ul class="uk-switcher uk-margin" id="item-tabs">
            <li>Item 1</li>
            <li>Item 2</li>
            <li>Item 3</li>
            <li>Item 4</li>
        </ul>
    </div>
</div>
```
When both of these items exist on the page at the same time, I get the following error: "ReferenceError: Can't find variable: UIKit".

When I remove one or the other, it goes away.

**Comment by saschadube** [comment]
Please provide a working CodePen http://codepen.io/anon/pen/KaZqdw

**Comment by qwertytech** [comment]
I've narrowed it down to the uk-tab-left class when I try to put the tab list on the left side of the content.  Here is the error I'm getting.
<img width="1390" alt="screenshot 2017-01-30 22 20 13" src="https://cloud.githubusercontent.com/assets/899016/22452203/72ab42e0-e73a-11e6-9147-6faa0e47debc.png">

**Comment by janschoenherr** [comment]
Fixed in 246320e28455721424a4aa910568594ab3176923
